hardhat.config.ts

```diff
@@ -1,143 +1,158 @@
+        blockNumber: BLOCK_NUMBER[CHAINID],
+        gasLimit: 8e6,
+      },
+    },
+    mainnet: {
+      url: process.env.TEST_URI,
+      chainId: 1,
+      accounts: {
+        mnemonic: process.env.MAINNET_MNEMONIC,
+      },
+    },
+    kovan: {
+      url: process.env.KOVAN_URI,
+      chainId: 42,
+      accounts: {
+        mnemonic: process.env.KOVAN_MNEMONIC,
+      },
+    },
+    bsc_mainnet: {
+      url: process.env.BSC_MAINNET_URI,
+      chainId: 56,
+      accounts: {
+        mnemonic: process.env.BSC_MAINNET_MNEMONIC,
+      },
+    },
+    avax: {
+      url: process.env.AVAX_URI,
+      chainId: 43114,
+      accounts: {
+        mnemonic: process.env.AVAX_MNEMONIC,
+      },
+    },
+    fuji: {
+      url: process.env.FUJI_URI,
+      chainId: 43113,
+      accounts: {
+        mnemonic: process.env.FUJI_MNEMONIC,
+      },
+    },
+  },
+  namedAccounts: {
+    deployer: {
+      default: 0,
+      1: "0xF8368119Bb1073Cf01B841848725d81b542A4c19",
+      42: "0x422f7Bb366608723c8fe61Ac6D923023dCCBC3d7",
+      56: "0xF8368119Bb1073Cf01B841848725d81b542A4c19",
+      43114: "0x422f7Bb366608723c8fe61Ac6D923023dCCBC3d7",
+      43113: "0x422f7Bb366608723c8fe61Ac6D923023dCCBC3d7",
+      1313161554: "0x46B4E6143Fb6ded2e5FBd87887Ef4f50f716dcA0",
+      1313161555: "0x46B4E6143Fb6ded2e5FBd87887Ef4f50f716dcA0",
+    },
+    owner: {
+      default: 0,
+      1: "0x77DA011d5314D80BE59e939c2f7EC2F702E1DCC4",
+      42: "0x92Dd37fbc36cB7260F0d2BD09F9672525a028fB8",
+      56: "0xF8368119Bb1073Cf01B841848725d81b542A4c19",
+      43114: "0x939cbb6BaBAad2b0533C2CACa8a4aFEc3ae06492",
+      43113: "0x004FCF8052D3c7eCb7558ac0068882425a055528",
+      1313161554: "0x46B4E6143Fb6ded2e5FBd87887Ef4f50f716dcA0",
+      1313161555: "0x46B4E6143Fb6ded2e5FBd87887Ef4f50f716dcA0",
+    },
+    keeper: {
+      default: 0,
+      1: "0x691c87dc570563D1D0AD7Fd0bb099ED367858863",
+      42: "0x691c87dc570563D1D0AD7Fd0bb099ED367858863",
+      56: "0xF8368119Bb1073Cf01B841848725d81b542A4c19",
+      43114: "0x422f7Bb366608723c8fe61Ac6D923023dCCBC3d7",
+      43113: "0x004FCF8052D3c7eCb7558ac0068882425a055528",
+      1313161554: "0xA4290C9EAe274c7A8FbC57A1E68AdC3E95E7C67e",
+      1313161555: "0xA4290C9EAe274c7A8FbC57A1E68AdC3E95E7C67e",
+    },
+    admin: {
+      default: 0,
+      1: "0x43a43D3404eaC5fA1ec4F4BB0879495D500e390b",
+      42: "0x422f7Bb366608723c8fe61Ac6D923023dCCBC3d7",
+      56: "0xF8368119Bb1073Cf01B841848725d81b542A4c19",
+      43114: "0x31351f2BD9e94813BCf0cA04B5E6e2b7ceAFC7c6",
+      43113: "0x004FCF8052D3c7eCb7558ac0068882425a055528",
+      1313161554: "0x46B4E6143Fb6ded2e5FBd87887Ef4f50f716dcA0",
+      1313161555: "0x46B4E6143Fb6ded2e5FBd87887Ef4f50f716dcA0",
+    },
+    feeRecipient: {
+      default: 0,
+      1: "0xDAEada3d210D2f45874724BeEa03C7d4BBD41674", // Ribbon DAO
+      42: "0x92Dd37fbc36cB7260F0d2BD09F9672525a028fB8",
+      56: "0xF8368119Bb1073Cf01B841848725d81b542A4c19",

```

Comment:
can we change this to 0xDAE?

Reply:
Addressed in https://github.com/ribbon-finance/ribbon-v2/pull/300/commits/e11a3cd85afed42c4bee272f44df8f9afe17bd31